### PR TITLE
feat(UI-945): trigger info popover on hover

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
User experience optimization and bringing the current feature (trigger info) to use the same UX patterns we have so far (after UI/UX feedback):
1. The info isn't action, it shouldn't be in the actions column, move to the trigger type column instead of the type icon
2. Information popovers are usually displayed on hover, not on click, change the popover behavior to hover, not click

## Linear Ticket
https://linear.app/autokitteh/issue/UI-945/triggers-info-popper

## What type of PR is this? (check all applicable)

-   [x] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/organisms/triggers/table/popoverContent.tsx [hunk 1]
```diff
@@ -3,107 +3,134 @@ import React from "react";
...
-							<CopyButton size="sm" text={trigger.eventType} />
 						</div>
 					) : null}
 					{trigger.filter ? (
 						<div className="flex items-center gap-x-1">
 							<div className="font-semibold">{t("filter")}:</div>
 							<div>{trigger.filter}</div>
-							<CopyButton size="sm" text={trigger.filter} />
 						</div>
 					) : null}
```
Comment (J1za) [inline src/components/organisms/triggers/table/popoverContent.tsx:135]:
```
const connectionDetails = [
  {
    label: t("connection"),
    value: triggerConnection?.name,
    icon: TriggerConnectionItegrationIcon
  },
  {
    label: t("connectionId"),
    value: triggerConnection?.connectionId,
  },
  { label: t("file"), value: trigger.path },
  { label: t("entrypoint"), value: trigger.entryFunction },
  { label: t("eventType"), value: trigger.eventType },
  { label: t("filter"), value: trigger.filter },
];
```

```
{connectionDetails.map(
      ({ label, value, icon: Icon }) =>
        value ? (
          <div key={label} className="flex items-center gap-x-1">
            <div className="font-semibold">{label}:</div>
            {Icon && <Icon className="size-4" />}
            {value}
          </div>
        ) : null
    )}
```

File: src/components/organisms/triggers/table/popoverContent.tsx [hunk 2]
```diff
@@ -3,107 +3,134 @@ import React from "react";
...
+							<div className="font-semibold">{t("file")}:</div>
+							{trigger.path}
+						</div>
+					) : null}
+					{trigger.entryFunction ? (
+						<div className="flex items-center gap-x-1">
+							<div className="font-semibold">{t("entrypoint")}:</div>
+							{trigger.entryFunction}
+						</div>
+					) : null}
```
Comment (J1za) [inline src/components/organisms/triggers/table/popoverContent.tsx:84]:
```
const scheduleDetails = [
  { label: t("cron"), value: trigger.schedule },
  { label: t("file"), value: trigger.path },
  { label: t("entrypoint"), value: trigger.entryFunction },
];
```
```
 {scheduleDetails.map(
      ({ label, value }) =>
        value ? (
          <div key={label} className="flex items-center gap-x-1">
            <div className="font-semibold">{label}:</div>
            <div>{value}</div>
          </div>
        ) : null
    )}
```

File: src/components/organisms/triggers/table/popoverContent.tsx [hunk 3]
```diff
@@ -3,107 +3,134 @@ import React from "react";
...
```
Comment (J1za) [inline src/components/organisms/triggers/table/popoverContent.tsx:55]:
```
const webhookDetails = [
  { label: t("file"), value: trigger.path },
  { label: t("entrypoint"), value: trigger.entryFunction },
];
```
```
{webhookDetails.map(
  ({ label, value }) =>
    value ? (
      <div key={label} className="flex items-center gap-x-1">
        <div className="font-semibold">{label}:</div>
        <div>{value}</div>
      </div>
    ) : null
)}
```

Comment (RonenMars) [reply src/components/organisms/triggers/table/popoverContent.tsx]: done

File: src/components/organisms/triggers/table/popoverContent.tsx (hunk 2 above)
Comment (RonenMars) [reply src/components/organisms/triggers/table/popoverContent.tsx]: done

File: src/components/organisms/triggers/table/popoverContent.tsx (hunk 1 above)
Comment (RonenMars) [reply src/components/organisms/triggers/table/popoverContent.tsx]: Done

**Review by J1za — APPROVED** [review]

**Pull request merged**